src/stories/components/Markdown/index.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react-perf/jsx-no-new-array-as-prop */
+
+import ReactMarkdown, { Options } from 'react-markdown';
+
+import gfm from 'remark-gfm';
+
+export const Markdown = ({ ...props }: Options) => {
+  return (
+    <>
+      <div>
+        <ReactMarkdown {...props} remarkPlugins={[gfm]} />
+      </div>
+    </>
```

Comment:
Acredito que neste caso pode remover o Fragment.

```suggestion
    <div>
      <ReactMarkdown {...props} remarkPlugins={[gfm]} />
    </div>
```